validation/kubernetes/eks-core/main.tf

```diff
@@ -88,7 +88,8 @@ module "eks_core" {
     volume_claim_storage_class_name = "default"
     volume_claim_size               = "5Gi"
 
-    alertmanager_enabled = false
+    alertmanager_enabled                 = false
+    secrets_store_csi_driver_aws_enabled = true
```

Comment:
I don't understand why this is set to `true` here, but the default in `eks-core/variables.tf` is false. I expect them to be the same but maybe I'm missing something.

Same comment applies for `secrets_store_csi_driver_azure_enabled` above.

Reply:
This is just validation config @ahockersten since it have a default value we don't even have to define it.